junit-jupiter-engine/src/test/java/org/junit/jupiter/engine/execution/ParameterResolutionUtilsTests.java

```diff
@@ -93,23 +92,23 @@ void constructorInjectionWithMissingResolver() {
 	}
 
 	@Test
-	void invokingMethodsWithoutParameterDoesNotDependOnParameterResolvers() {
+	void resolvingArgumentsForMethodsWithoutParameterDoesNotDependOnParameterResolvers() {
 		testMethodWithNoParameters();
 		throwDuringParameterResolution(new RuntimeException("boom!"));
 
-		invokeMethod();
+		Object[] arguments = resolveMethodParameters();
 
-		verify(instance).noParameter();
+		assertThat(arguments).hasSize(0);
```

Comment:
Does `isEmpty()` exist for array assertions in AssertJ?

Reply:
Yes :+1: